patches/components-browser_ui-site_settings-android-website_preference_bridge.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/browser_ui/site_settings/android/website_preference_bridge.cc b/components/browser_ui/site_settings/android/website_preference_bridge.cc
+index a5316d02726f1..0a587c71d8268 100644
+--- a/components/browser_ui/site_settings/android/website_preference_bridge.cc
++++ b/components/browser_ui/site_settings/android/website_preference_bridge.cc
+@@ -813,6 +813,7 @@ static void JNI_WebsitePreferenceBridge_SetContentSettingEnabled(
+       case ContentSettingsType::POPUPS:
+       case ContentSettingsType::SENSORS:
+       case ContentSettingsType::SOUND:
++      case ContentSettingsType::AUTOPLAY:
```

Comment:
this can be done with a chromium src override
`define AUTOMATIC_DOWNLOADS ContentSettingsType::AUTOPLAY: value = CONTENT_SETTINGS_ALLOW; break; AUTOMATIC_DOWNLOADS`

Reply:
Doesn't that still require a patch? Or is it for maintainability in the future?

Reply:
it does not require a patch and it is extensible unlike this patch. Even if we patched, we wouldn't patch like this because this is not extensible and does not follow the patching guidelines